src/Shipment/TaxIdentificationNumbers/TaxIdentificationNumbers.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelCom\Integration\Shipment\TaxIdentificationNumbers;
+
+use ArrayObject;
+use function array_map;
+
+class TaxIdentificationNumbers extends ArrayObject
+{
+    public function __construct(TaxIdentificationNumber ...$taxIdentificationNumbers)
+    {
+        parent::__construct($taxIdentificationNumbers);
+    }
+
+    public function toArray(): array
+    {
+        return array_map(static fn(TaxIdentificationNumber $taxIdentificationNumber) => $taxIdentificationNumber->toArray(), (array) $this);
```

Comment:
Perhaps in this simple context we can use a shorter variable name - up to you, though. I usually prefer to use something like `$number` since there no other variables to pollute the semantics.
```suggestion
        return array_map(static fn(TaxIdentificationNumber $number) => $number->toArray(), (array) $this);
```